graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/buffers/MessageEvent.java

```diff
@@ -45,6 +46,7 @@ public MessageEvent newInstance()
 
     private RawMessage raw;
     private Message msg;
```

Comment:
It would probably make sense to get rid of the single message field, express `getMessage()` and `setMessage()` as delegates to `getMessageList()` and `setMessageList()`, and mark the singular methods as deprecated.